ansible/playbooks/roles/kubernetes_master/templates/kubeadm-config.yml.j2

```
@@ -24,6 +24,12 @@ apiServer:
 {% for key, value in specification.advanced.api_server_args.items() %}
     {{ key }}: "{{ value }}"
 {% endfor %}
+  extraVolumes:
+    - name: apiserver-log
+      hostPath: /var/log/apiserver
+      mountPath: /var/log/apiserver
```

```suggestion
  extraVolumes:
    - name: audit-log
      hostPath: /var/log/kubernetes/audit/
      mountPath: /var/log/kubernetes/audit/
```
Paths like in docs: https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend

---

This will change our previous default values, I would keep it as it was before, not to patch anything on upgrade. WDYT?

---

Decided to leave the path in container as it was before, but make it non-configurable - doesn't make sense without configurable extraVolumes in kubeadm config. Host path is changed as suggested.